Description: Support for eurosym package
 This patch adds support for the eurosymbol package, i.e. the \euro and \EUR{}
 macros.
Author: Jorgen Schaefer <forcer@debian.org>
Author: Roland Stigge <stigge@antcom.de>
Bug-Debian: http://bugs.debian.org/163479

--- /dev/null
+++ latex2html-2008-debian1/styles/eurosym.perl
@@ -0,0 +1,5 @@
+sub do_cmd_euro{join('', "&euro;", $_[0]);}
+sub do_cmd_EUR{join('', "&euro;", $_[0]);}
+
+1; # last line
+
